Makefile

```
@@ -7,27 +7,39 @@ CONTAINER_ENGINE?=$(shell command -v podman 2>/dev/null || echo "docker")
 unexport REPO_NAME
 
 .PHONY: isclean
-isclean:
+isclean: ## Validate the local checkout is clean. Use ALLOW_DIRTY_CHECKOUT=true to nullify
 	@(test "$(ALLOW_DIRTY_CHECKOUT)" != "false" || test 0 -eq $$(git status --porcelain | wc -l)) || (echo "Local git checkout is not clean, commit changes and try again." >&2 && exit 1)
 
 .PHONY: tag-check
-tag-check:
+tag-check: ## Perform a tag-check that validates a new tag has been created when changing the build image
 	config/tag-check.sh
 
 .PHONY: test
-test: isclean tag-check
+test: ## Runs tests under the /case directory
+	isclean tag-check
 	test/driver $(CASE_GLOB)
 
 .PHONY: pr-check
-pr-check: test
+pr-check: ## This is the target ran by prow
+	test
```

Okay, I see the problem. The chained targets need to remain on the line with the target name.
```suggestion
pr-check: test ## This is the target ran by prow
```